src/libsystemd-network/dhcp-network.c

```diff
@@ -113,6 +115,13 @@ static int _bind_raw_socket(
         if (r < 0)
                 return -errno;
 
+        if (so_priority_set)
+        {
```

Comment:
coding style, please put `{` on the same line as `if`, like we do everywhere else